docs/guides/cd.md

```diff
@@ -1,64 +1,64 @@
 # Continuous Deployment of your Functions
 
-To continously deploy your functions on TriggerMesh you can register a source code repository which contains your function code and a manifest (`serverless.yaml`).
 
-The **_Repositories_** section in the console contains a view to register any repository, from previously linked accounts, for continuous deployment.
 
+The Triggermesh console provides an interface called [Repositories.](https://cloud.triggermesh.io/function-triggers) Here the user is able to select a codebase, from a linked repository provider, to be continuously deployed. This codebase **must** contain a deployment manifest[^1] . 
 
-## Register the repository
 
+## Register the repository
 
-**Begin by forking the [sample repository](https://github.com/sebgoa/tmserverless) which contains a function and an example `serverless.yaml` manifest.**
+###### The following example uses this [sample repository.](https://github.com/sebgoa/tmserverless) To follow along begin by forking the example to your repository provider. 
```

Comment:
Why go with the sub-level vs keeping the bold typesetting?

Reply:
This is more of a note or an aside then a statement. I wanted it to be small and its appearance to reflect its intention as a side note. If you think we should move it back to bold or regular i have no issues

Reply:
Fair enough.  Although addressed on slack, for others who may see this later, the heading indicator could be parsed to automatically generate things like a table of contents and would not be a good use here.  My personal convention is use italics for notes and bold for important items.